src/EnergyPlus/PollutionModule.hh

```diff
@@ -446,1139 +324,50 @@ namespace PollutionModule {
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0),
-        PollutionModule::CoefficientProps(static_cast<int>(PollutionModule::PollFactor::OtherFuel2),
-                                          false,
-                                          1.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0),
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0};
-    PollutionModule::FuelTypeProps FuelType;
+    std::array<Pollution::ComponentProps, (int)Pollution::PollFuelComponent::Num> pollComps;
+
+    // Meters, meter values, and grouped meter values
+    std::array<int, (int)Pollution::PollFacilityMeter::Num> facilityMeterNums = {-1};
+    std::array<Real64, (int)Pollution::PollFacilityMeter::Num> facilityMeterVals = {0.0};
+    std::array<Real64, (int)Pollution::PollFuelComponent::Num> facilityMeterFuelComponentVals = {0.0};
+
+    std::array<Real64, (int)Pollution::Pollutant::Num> pollutantVals = {0.0};
+
+    std::vector<Pollution::PollFuel> pollFuelFactorList;
```

Comment:
This vector is here because the order in which some output is printed matches the order in which `FuelFactors` objects appear in the IDF file, and this helps eliminate order dependent diffs.

Reply:
Understood, not necessarily a functional change, but helping the review process.  Do you anticipate removing this little band-aid in its own PR?  It will cause lots of diffs, but if it's in the context of a tiny code change, that's easy to handle.

Reply:
It doesn't matter to me. Now that it's there it can stay.